build-logic/settings.gradle.kts

```diff
@@ -10,86 +7,10 @@ include("checks")
 include("publication")
 
 pluginManagement {
-    val artifactoryUrl: String? by settings
-
-    fun MavenArtifactRepository.artifactoryUrl(repositoryName: String) {
-        setUrl("$artifactoryUrl/$repositoryName")
-        isAllowInsecureProtocol = true
-    }
-
-    fun MavenArtifactRepository.setUrlOrProxy(repositoryName: String, originalRepo: String) {
-        if (artifactoryUrl.isNullOrBlank()) {
-            name = repositoryName
-            setUrl(originalRepo)
-        } else {
-            name = "Proxy for $repositoryName: $originalRepo"
-            artifactoryUrl(repositoryName)
-        }
-    }
-
-    repositories {
-        maven {
-            setUrlOrProxy("gradle-plugins", "https://plugins.gradle.org/m2/")
-        }
-    }
-}
-
-val artifactoryUrl: String? by settings
-
-fun MavenArtifactRepository.artifactoryUrl(repositoryName: String) {
-    setUrl("$artifactoryUrl/$repositoryName")
-    isAllowInsecureProtocol = true
-}
-
-fun MavenArtifactRepository.setUrlOrProxy(repositoryName: String, originalRepo: String) {
-    if (artifactoryUrl.isNullOrBlank()) {
-        name = repositoryName
-        setUrl(originalRepo)
-    } else {
-        name = "Proxy for $repositoryName: $originalRepo"
-        artifactoryUrl(repositoryName)
-    }
+    includeBuild("../build-logic-settings")
```

Comment:
Now I see when it's useful.
In Gradle release notes didn't pay attention to it.